src/store/common/api-clients.ts

```diff
@@ -0,0 +1,58 @@
+import { selector } from 'recoil';
+import { currentNetworkState } from '@store/networks';
+import {
+  Configuration,
+  AccountsApi,
+  SmartContractsApi,
+  InfoApi,
+  BlocksApi,
+} from '@stacks/blockchain-api-client';
+import { fetcher } from '@common/api/wrapped-fetch';
+
+enum KEYS {
```

Comment:
Would suggest not repeating Keys name, using something more specific. Would be helpful when searching